holoviews/plotting/plot.py

```diff
@@ -258,6 +260,15 @@ def traverse(self, fn=None, specs=None, full_breadth=True):
                 if not full_breadth: break
         return accumulator
 
+    @property
+    def force(self):
+        return self._force
+
+
+    @force.setter
+    def force(self, value):
+        self.traverse(lambda x: setattr(x, '_force', value))
+
```

Comment:
From what I see below, I think having ``self.force`` be an attribute without ``self._force`` would be simpler (i.e no properties). The reason is that in the context_manager you just need to access``self.force``.  Then you set ``self.force`` to True and False in one place in this file. As a result, I'm not sure it is worth introducing properties and an underscore attribute when you can just inline two fairly short traversal expressions.